### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by platipo** [silverwind/droppy#269]
Droppy would become more friendly if users could use it in their own language.

[l10n.js](https://github.com/eligrey/l10n.js/)  could be suitable for this purpose.

**Comment by silverwind** [comment]
While the idea is good, I'm not sure I want to include a additional client library. A better solution would probably be to have localization built into the Handlebars templates, possibly with a new helper function.

I first need to turn most of the content in `client/html` into templates for that to work.

**Comment by platipo** [comment]
Wow! Never heard of Handlebars templates, it seems astonishing! 


> new helper function

You mean like this https://gist.github.com/davidwkeith/3093861

**Comment by silverwind** [comment]
Yes similar, but simpler. It would just be a lookup on a JavaScript object for a localization string, so you can do

```html
<p>{{{loc "some english string"}}}</p>
```

and it would use the translated version depending on the client browser's `Accept-Language` preferences. The same function can then be used from JS too. Pretty similar to what I have set up for SVG images right now.

**Comment by platipo** [comment]
Even if I'd like to help and the task seems easy, I'm a js noob :sob:

**Comment by silverwind** [comment]
You can contribute translations later ;)
